articles/machine-learning-and-deep-learning/index.md

```diff
@@ -0,0 +1,121 @@
+
+The AI incorporates and avails intelligence that is used to enhance decision making on both applications. These applications have contributed significantly to solving problems that previously affected performances in some applications.  
+
+New ideas in the tech world have led to various innovations that have sparked the nature of things. Alexa and Siri are [virtual assistant](https://www.investopedia.com/terms/v/virtual-assistant.asp) that enhance the use of machine learning and are widely used.
+
+Machine learning and deep learning have attracted significant attention over the years. The popularity of these two has led to various applications implementing its tools.
+
+This article will focus on understanding what machine learning and deep learning are. The reader will have a deeper understanding of how these applications differ and enhance their functionalities.
+
+### Machine Learning and Artificial Intelligence
+
+Machine learning is a [subset](https://en.wikipedia.org/wiki/Subset) of AI that allows computer algorithms to make accurate data outcomes through interpretations.
+
+Machine learning is used to obtain and use data that is used to make final decisions. Machine learning processes are also dependent on the subsets of artificial intelligence.
+
+Machine learning allows the use the AI towards automating several functionalities used by machine learning. The AI enhances neuroscience algorithm that emulates human intelligence, that is able to make more accurate predictions. The neuorscience algorithm is modified in a way that emulates the human intelligence. This is because it is able to understand various relationships between data, and make predictions that are accurate.  
+
+Machine learning adopts the use of several [synthetic intelligence](https://www.wikiwand.com/en/Synthetic_intelligence) aspects and modern [virtual assistants](https://www.investopedia.com/terms/v/virtual-assistant.asp) to enhance automation. The application of live chatbots is perfect example how web applications enhance communication. 
+
+Machine learning uses Artificial Intelligence (AI) techniques that assist in decision making. These techniques include the use of [conventional algorithms](http://wiki.cas.mcmaster.ca/index.php/Conventional_Encryption_Algorithms) used in performing mathematical computation and predictions.
+
+ Machine learning uses both structured and unstructured data to fetch data in applications. Structured data is presented in the form of columns and rows. Examples of structured data include strings, dates and numbers.  Unstructured data information is stored as files. Examples include; images, word documents, email files, videos and audio files.
+ 
+ The use of Support Vector Machines (SVMs) and decision-trees are examples that use unstructured data to make interpretations in machine learning.
```

Comment:
This is not required. I had mentioned it in the review as a note. Probably add more information to enhance the flow.